spec/vorpal/integration/db_driver_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'integration_spec_helper'
+require 'vorpal'
+
+describe Vorpal::DbDriver do
+  describe '#build_db_class' do
+    let(:db_class) { subject.build_db_class('trees') }
+
+    it 'generates a vald class name so that rails auto-reloading works' do
+      expect { Vorpal.const_defined?(db_class.name) }.to_not raise_error
+    end
+
+    it 'does not let the user access the generated class' do
+      expect { Vorpal.const_get(db_class.name) }.to raise_error(NameError)
+    end
+  end
+end
```

Comment:
Question: if you have a blank DB, does this test fail? I feel like you need one of these:

```ruby
  before(:all) do
    define_table('db_driver_test_tables')
  end
```

And not use the 'trees' table, since that is an implicit dependency on the setup done in https://github.com/nulogy/vorpal/blob/master/spec/vorpal/acceptance/aggregate_mapper_spec.rb